### PR TITLE
Improve log document

### PR DESCRIPTION
**Pull request by wxiaoguang**
Gitea's logger system is **very unfriendly** to users.

This PR improves the document slightly to make users can get their logs more easily.

### CODE REVIEW

File: docs/content/doc/help/seek-help.en-us.md [hunk 1]
```diff
@@ -20,10 +20,25 @@ menu:
 
 **NOTE:** When asking for support, it may be a good idea to have the following available so that the person helping has all the info they need:
 
-1. Your `app.ini` (with any sensitive data scrubbed as necessary)
-2. The `gitea.log` (and any other appropriate log files for the situation)
-    * e.g. If the error is related to the database, the `xorm.log` might be helpful
-3. Any error messages you are seeing
+1. Your `app.ini` (with any sensitive data scrubbed as necessary).
+2. The Gitea logs, and any other appropriate log files for the situation.
+    * The logs may be outputted to console by default. If you need to collect logs from files,
```
Comment (Gusted) [inline docs/content/doc/help/seek-help.en-us.md]: It may be a default? That doesn't sound like a default setting 😄.

**Review by Gusted — COMMENTED** [review]
Otherwise LGT M

File: docs/content/doc/help/seek-help.en-us.md (hunk 1 above)
Comment (wxiaoguang) [reply docs/content/doc/help/seek-help.en-us.md]: It depends on how the `app.ini` is generated.

Comment (Gusted) [reply docs/content/doc/help/seek-help.en-us.md]: I think we can assume it's logged to the console one way or another. Most would hopefully be running in docker or via systemd in which they use existing configurations which should have it enabled.

Comment (wxiaoguang) [reply docs/content/doc/help/seek-help.en-us.md]: Sorry but I didn't get the point of your comment. What should be done?

Comment (Gusted) [reply docs/content/doc/help/seek-help.en-us.md]: It sounds confusing to me, something like:
```suggestion
    * The logs are likely to be outputted to console. If you need to collect logs from files, 
```
Sounds quite more clear IMO.

Comment (wxiaoguang) [reply docs/content/doc/help/seek-help.en-us.md]: It is `MODE=console` by default , in most cases. But some setup steps can overwrite it.

image

Comment (Gusted) [reply docs/content/doc/help/seek-help.en-us.md]: I'm going to assume general case and those that have overwritten it are probably knowledgeable enough to understand that.

Comment (wxiaoguang) [reply docs/content/doc/help/seek-help.en-us.md]: Nope, they may not have such knowledge, for example, overwritten by docker environment, or some configurations generated by helmchart, or downstream packagers.

**Review by Gusted — APPROVED** [review]

File: docs/content/doc/help/seek-help.en-us.md [hunk 2]
```diff
@@ -20,10 +20,25 @@ menu:
...
-2. The `gitea.log` (and any other appropriate log files for the situation)
-    * e.g. If the error is related to the database, the `xorm.log` might be helpful
-3. Any error messages you are seeing
+1. Your `app.ini` (with any sensitive data scrubbed as necessary).
+2. The Gitea logs, and any other appropriate log files for the situation.
+    * The logs may be outputted to console by default. If you need to collect logs from files, 
+      you could copy the following config into your `app.ini` (remove all other `[log]` sections),
+      then you can find the `gitea.log` in Gitea's log directory (default: `%(GITEA_WORK_DIR)/log`).
+    ```
+    ; To show all SQLs, you could also set LOG_SQL=true in [database] section
```
Comment (delvh) [inline docs/content/doc/help/seek-help.en-us.md]:
```suggestion
    ```ini
    ; To show all SQL logs, you can also set LOG_SQL=true in the [database] section 
```

**Review by delvh — APPROVED** [review]

**Pull request merged**